node/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/certusone/wormhole/node
 
-go 1.20
+go 1.21
+
+toolchain go1.21.8
```

Comment:
1.21.9 or newer as per `docs/operations.md`

Reply:
I see now a helper script is being introduced to make it easier to bump Go versions 👏🏻 🙇🏻 this could be covered too.

Reply:
I tend to leave changes to the go.mod and go.sum files to the go command only, though I can do `go mod edit` ... to the latest version in that script. What do you think?

Reply:
Whatever makes you happy, I'm good.

Reply:
Fixed in 3e190a33dd9f6935fa8afe282643bab6a5835eff and 91a958475168281cd0a435a2ab30fe5c14393e54